test/test_all.rb

```diff
@@ -1,3 +1,6 @@
 Dir.chdir(File.dirname(__FILE__)) do
-  Dir['**/test_*.rb'].each { |file| require(file) }
-end
\ No newline at end of file
+  test_files = Dir['**/test_*.rb']
```

Comment:
I think you can write `(Dir['**/test_*.rb']-["test_all.rb"]).each(...)`

Reply:
Fixed